### PR TITLE
Add Styling

### PR DESCRIPTION
**Pull request by faisalakb**
In this project apply the following features : :100: 
 - Add any missing JSX elements :+1: 
 - Final design according to Zeplin's design :+1: 
 - Use the CSS rules detailed in Zeplin to style the website.  :+1: 
 - React setup . :+1: 
 - Setup Linters . :+1:

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -44,9 +45,9 @@ ul {
 }
```
Comment (Alfredbis29) [inline src/App.css:156]: - [ ] While I appreciate the effort that has gone into it, I must express my concern that the final design doesn't quite align with the original design concept provided at the outset of the project. I can see that your book title doesn't look the same as the Zeplin: the fonts and colors do not look the same as the design, kindly follow the design.
**Your Design**
Screenshot from 2023-09-01 18-13-50

**Zeplin Design**
Screenshot from 2023-09-01 18-14-24

File: src/App.css [hunk 2]
```diff
@@ -76,20 +77,18 @@ ul {
   display: flex;
   align-items: center;
   justify-content: center;
+  margin-left: 40%;
 }
```
Comment (Alfredbis29) [inline src/App.css:200]: - [ ] I've noticed that the header design doesn't quite match the original concept. Can you adjustments to bring it closer to the initial vision of it :upside_down_face: 
Your design 
Screenshot from 2023-09-01 18-31-10

The zeplin design
Screenshot from 2023-09-01 18-29-42

File: src/App.css [hunk 3]
```diff
@@ -163,3 +162,16 @@ ul {
 .removeButton {
```
Comment (Alfredbis29) [inline src/App.css:374]: - [ ] I was not able to inline on the particular part where you designed this section add the new book, kindly follow the design guides, if we compare your design again here are not really similar :+1: 
 
Your Design 
Screenshot from 2023-09-01 18-40-57

Zeplin Design to follow 
Screenshot from 2023-09-01 18-41-51

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @faisalakb  
 Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
:heavy_check_mark:  Your Work looks Professional 
:heavy_check_mark:  Nice work so far :+1: 
 
 ## Required Changes 

- [ ]  Check Under the review

 ## Optional suggestions 

- [ ] N/A
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 4]
```diff
@@ -123,43 +282,113 @@ ul {
...
+.MainInp {
+  margin-left: 8.8rem;
 }
 
 .inputContainer {
   display: flex;
   flex-direction: row;
   column-gap: 18%;
-  margin-left: 3%;
 }
```
Comment (Alusp) [inline src/App.css]: - [ ] Good job in implementing the changes requested by the previous reviewer, however, according to the previous reviewer, you should ensure that your design matches the given requirement. as it is right now the contents of your `.inputContainer` class is overlapping each other, kindly align the margin properties of it contents.  Matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustration of yours and the given Design, please pay attention on the red markings on your design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

Student


</td>
      <td>

studesign

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @faisalakb, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descrptive ✔️
- [ ] Professional Github Flow used ✔️
- [ ] Professional PR Summary Flow used ✔️
- [ ] UI Closer to the given design :heavy_check_mark:  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by faisalakb** [comment]
Please check, i updated the required changes said by the above reviewer.

File: src/component/BookDisplay.jsx [hunk 5]
```diff
@@ -19,19 +39,19 @@ const BookDisplay = () => {
...
-      // Dispatch addBookdata and handle the fetchBooks action in the .then() block
       dispatch(addBookdata(newBook))
         .then(() => {
           setBookTitle('');
-          dispatch(fetchBooks()); // Fetch the updated list of books
+          setSelectedType('');
+          dispatch(fetchBooks());
         })
         .catch((error) => {
           console.error('Error adding book:', error);
```
Comment (Johnadibe) [inline src/component/BookDisplay.jsx:57]: > - [OPTIONAL] It seems this wasn't noticed by the previous reviewer. I suggest you remove the console.error in your code, it makes your code look messy.😞

File: src/App.css [hunk 6]
```diff
@@ -123,43 +282,110 @@ ul {
...
+  margin: 1.188rem 0 0 2.125rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
   border-radius: 3px;
   background-color: #0290ff;
   color: white;
   font-weight: 400;
   font-size: medium;
-  border: none; /* Remove button border */
+  border: none;
+}
```
Comment (Johnadibe) [inline src/App.css:322]: - [OPTIONAL] It seems the size of your button design doesn't align with the provided design. Please kindly ensure to fix that.👍

btn

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @faisalakb👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Implemented the changes requested by the previous reviewer✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by faisalakb** [comment]
Thanks for the time and suggestions.

**Pull request merged**